assets/shared/helpers/player.js

```diff
@@ -0,0 +1,539 @@
+const VIDEO_TYPE = 'video';
+const VIDEOPRESS_TYPE = 'videopress';
+const YOUTUBE_TYPE = 'youtube';
+const VIMEO_TYPE = 'vimeo';
+
+const players = {
```

Comment:
This is the players "gateway" to abstract the API of each one. I thought sometimes to extract it to a different file, but for now I decided to keep it in a single file. Mainly to not create an extra folder in the `helpers`, but we can rethink it as it grows in the future (or now if we don't agree).

Reply:
> we can rethink it as it grows in the future (or now if we don't agree)

Just saying here that I'm already re-thinking it as part of the next PR that is coming to handle players in the editor. 😉